packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -660,6 +662,13 @@ export interface CallAdapterCallOperations {
    * @public
    */
   updateSelectedVideoBackgroundEffect(selectedVideoBackground: VideoBackgroundEffect): void;
+  /* @conditional-compile-remove(end-of-call-survey) */
+  /**
+   * Send the end of call survey result
+   *
+   * @beta
+   */
+  submitSurvey(survey: CallSurvey, options?: SubmitSurveyOptions): Promise<CallSurveyResponse | undefined>;
```

Comment:
when does it return undefined?

Reply:
Not sure but this is how it's defined on calling side. Should  we follow their definition?